src/azure-cli/HISTORY.rst

```diff
@@ -18,6 +18,7 @@ az webapp create : Fixing issue when running the command with --runtime
 **ACS**
 
 * aks create/update: add `--vnet-subnet-id` validation
+* aks create: add `--enable-private-cluster`
```

Comment:
I think we should have separate section for aks, like:
**AKS**
* aks create: add `--enable-private-clister`

Reply:
tests are added in `TestPopulateApiServerAccessProfile`

Reply:
**ACS** is actually "AKS", our module name is "acs" in Azure CLI

Reply:
let me rename it to AKS anyway.

Reply:
The history note is added in a released version 2.1.0. Please delete it and use PR title or PR description, they will be included in history.rst before release. You can check the guide [here](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

Reply:
The history note is added in a released version 2.1.0. You don't need to manually add history now, please delete it and use PR title or PR description, they will be included in history.rst before release. You can check the guide [here](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

Reply:
OK, let me delete it

Reply:
@fengzhou-msft deleted the changes in history. PTAL.

Reply:
Ok. I added the history note in description since current PR title has different wordings.